File: java/eneProject/src/org/nju/ene/pojo/EneParameter.java
package org.nju.ene.pojo;

public class EneParameter {
	//股票代号
	private String code;
	//ENE指标上限
	private int upper;
	//ENE指标下限
	private int lower;
	//ENE天数
	private int days;
	
	public int getUpper() {
		return upper;
	}
	public void setUpper(int upper) {
		this.upper = upper;
	}
	public int getLower() {
		return lower;
	}
	public void setLower(int lower) {
		this.lower = lower;
	}
	public int getDays() {
		return days;
	}
	public void setDays(int days) {
		this.days = days;
	}
	public String getCode() {
		return code;
	}
	public void setCode(String code) {
		this.code = code;
	}
}

File: python/ENEPro/EneParameter.py
# encoding: UTF-8
class EneParameter:
    #股票代号
    #code=''
    #ENE指标上限
    #upper=0
    #ENE指标下限
    #lower=0
    #ENE天数
    #days=0
    
    def getUpper(self) :
        return self.upper
    
    def setUpper(self,upper) :
        self.upper = upper
    
    def getLower(self) :
        return self.lower
    
    def setLower(self,lower) :
        self.lower = lower
    
    def getDays(self) :
        return self.days
    
    def setDays(self,days) :
        self.days = days
    
    def getCode(self) :
        return self.code
    
    def setCode(self,code) :
        self.code = code
    